projectroles/views.py

```diff
@@ -2964,6 +2992,23 @@ class RemoteSiteDeleteView(
     slug_field = 'sodar_uuid'
 
     def get_success_url(self):
+        """Override get_success_url() to add message"""
+        timeline = get_backend_api('timeline_backend')
+        if timeline:
+            event_name = (
+                'source_site_delete'
+                if self.object.mode == SITE_MODE_SOURCE
+                else 'target_site_delete'
+            )
+            description = 'delete remote site "{}"'.format(self.object.name)
+            timeline.add_event(
```

Comment:
Status also missing from here, so we get events with a perpetual `INIT` status. Also check other added events for the same issue.